Platform/OpenCore/OpenCoreMisc.c

```diff
@@ -159,6 +159,59 @@ OcToolDescribeEntry (
   HasIcon  = FALSE;
   HasLabel = FALSE;
 
+  if (ChosenEntry->Type == OC_BOOT_SYSTEM) {
+
+    if (StrCmp(ChosenEntry->Name, OC_MENU_RESET_NVRAM_ENTRY) == 0) {
```

Comment:
Hmm, maybe we should have a secondary identifier? I really do not like the idea of using name for this.
cc @vit9696

Reply:
Yes, that's what I asked @usr-sse2 to do for now when he asked me that it is not right. Maybe indeed turn system into reset nvram and then add another type when we need?

Reply:
Depends, how many System entries could we possibly have? Either this or a secondary type